release_notes/ocp_3_11_release_notes.adoc

```diff
@@ -2333,3 +2333,213 @@ To upgrade an existing {product-title} 3.10 or 3.11 cluster to this latest
+assist in troubleshooting CRD errors.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1702693[*BZ#1702693*])
+
+* There was a missing `@` for an instance variable in the Fluentd remote
+syslog plugin code. In some cases, systemd-journald logged errant values.
+This resulted in rsyslog forwarding failures. Now, the variable has been
+corrected remote logging completes successfully.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1703904[*BZ#1703904*])
+
+* Long running Jenkins agent and slave pods would experience defunct
+process errors, causing a high number of processes to appear in process
+listings until the pod is terminated. Now, `dumb-init` is deployed to clean
+up these defunct processes.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1707448[*BZ#1707448*])
+
+* The environment variable JOURNAL_READ_FROM_HEAD was set to an empty
+string.  This caused the default value of `read_from_head` for the journald
+input to be true.  When Fluentd starts up for the first time on a node, it
+reads in the entire journal. This could result in hours of delays for
+system messages to show up in ElasticSearch and Kibana. Now, Fluentd will
+check if the value is set and is not empty, or will use the default value
+of false. Fluentd will read from the tail of the journal when it starts on
+a new node.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1707524[*BZ#1707524*])
+
+* The script 99-origin-dns.sh had a debug flag set to enabled, which would
+log debug level messages by default. This has been resolved and debug is
+now set to false.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1707799[*BZ#1707799*])
+
+* Kubernetes pod templates were removed at random. This was because the OpenShift
+Jenkins Sync plugin confused ImagesStreams and ConfigMaps with the sane name
+while processing them. An event for one type could delete the pod template
+created for another type. The plugin has been modified to keep track of which
+api object type created the pod template of a given name.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1709626[*BZ#1709626*])
+
+* The `openshift_set_node_ip` variable was deprecated, but still included
+in inventory example files. This has now been removed from example files
+and code for the `openshift_set_node_ip` variable has been cleaned up.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1712488[*BZ#1712488*])
+
+* Previously, the web console could show an incorrect "Scaling to..." value for
+stateful sets in the project overview under some conditions. The stateful set
+desired replicas value now correctly updates in the web console project
+overview.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1713211[*BZ#1713211*])
+
+* Previously, a service would not correctly show up in the project overview when
+it selected the `deploymentconfig` label that is automatically set for pods
+created by a deployment config. Now, it correctly show services that select the
```

Comment:
Again on being consistent with how deploymentconfig/DeploymentConfig/deployment config is used in 3.x.

Reply:
Fixed all lower case "deploymentconfig" to "DeploymentConfig".